packages/assets-controllers/src/Standards/ERC20Standard.ts

```diff
@@ -55,16 +56,10 @@ export class ERC20Standard {
   async getTokenSymbol(address: string): Promise<string> {
     // Signature for calling `symbol()`
     const payload = { to: address, data: '0x95d89b41' };
-    const result = await this.provider.call(payload);
-
-    const abiCoder = new AbiCoder();
-
+    const result = (await this.provider.call(payload)) as `0x${string}`;
     // Parse as string - treat empty string as failure
     try {
-      const decoded = abiCoder.decode(['string'], result)[0];
-      if (decoded?.length > 0) {
-        return decoded;
-      }
+      return decodeSingle('string', result) as string;
```

Comment:
The return type should be inferred automatically here.

```ts
const foo = decodeSingle('string', ...);
type Foo = typeof foo; // string
```

Reply:
removed type casting [here](https://github.com/MetaMask/controllers/pull/985/commits/aec546b4268ff0fd2e72bea64f8295b6b5d1d2ce)